src/OrchardCore/OrchardCore.Mvc.Core/Utilities/StringExtensions.cs

```diff
@@ -14,19 +15,26 @@ public static class StringExtensions
     {
         public static string CamelFriendly(this string camel)
         {
+            // optimize common cases
             if (string.IsNullOrWhiteSpace(camel))
             {
                 return "";
             }
 
-            var sb = new StringBuilder(camel);
+            if (camel == "RawHtml")
```

Comment:
Do you where this one is coming from exactly?

Reply:
@deanmarcussen found it, it's from the Widget name that is included as a class in the default template. That's border-line. An mru cache would be better in that case since it's only because you used The Agency template apparently, and each site will have different common cases.

Reply:
OK, I don't start the dance with MRU as it will need to have locking etc, so removed the special case and now always falls to building the string, should be quite good as ZString will use array pool anyway.